editor/editor_node.cpp

```diff
@@ -6667,7 +6668,42 @@ void EditorNode::_add_renderer_entry(const String &p_renderer_name, bool p_mark_
 		item_text = TTR("Compatibility");
 	}
 	if (p_mark_overridden) {
-		item_text += " " + TTR("(Overridden)");
+		item_text += " - " + TTR("Overridden");
+	}
+
+	// Append short driver name to the renderer name, so that it's visible on screenshots of the editor.
+	String driver_name_short;
+	if (p_renderer_name != "gl_compatibility") {
+		String driver_name = GLOBAL_GET("rendering/rendering_device/driver");
+		if (RS::get_singleton()->get_current_rendering_method() != "gl_compatibility") {
+			// Take CLI argument overrides and automatic fallbacks into account.
+			driver_name = RS::get_singleton()->get_current_rendering_driver_name();
+		}
+		if (driver_name == "vulkan") {
+			driver_name_short = "VK";
+		} else if (driver_name == "d3d12") {
+			driver_name_short = "D3D";
+		} else if (driver_name == "metal") {
+			driver_name_short = "MT";
```

Comment:
```suggestion
			driver_name_short = "D3D12";
		} else if (driver_name == "metal") {
			driver_name_short = "METAL";
```
If "GLES" and "ANGLE" are acceptable short names, these seem like they could be 5 characters too?

Reply:
Or at least `MTL` (which is used as a Metal function prefix). But if 5 chars are OK, 6 is ok as well, and we should show full name ("Vulkan", "D3D12", "Metal").